src/makeCalculator.js

```diff
@@ -37,7 +37,72 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const allOperates = {
+
+    // змінна в якій буде зберігатись результат
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming
Fix all cases please

Reply:
Still not fixed from the previous review, need to fix it everywhere
<img width="897" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/64750958/008bc4f4-edb5-4416-b076-ca12caf38f95">